src/SFA.DAS.ToolsNotifications.Api/appsettings.json

```diff
@@ -5,5 +5,13 @@
     }
   },
   "AllowedHosts": "*",
-  "RedisConnectionString": ""
+  "APPINSIGHTS_INSTRUMENTATIONKEY": "",
+  "RedisConnectionString": "",
+  "LoggingRedisConnectionString": "",
+  "EnvironmentName": "LOCAL",
+  "ApiName": "Tool Service Notification API",
+  "PathBase": "/api/notifications",
+  "AzureAdTenantId": "",
+  "AzureADResourceId": "",
+  "AzureADClientId": ""
```

Comment:
Any of these that are environment specific need adding into the deployment manifest